doc/source/cluster/index.rst

```diff
@@ -24,3 +28,8 @@ Autoscaler uses a simple binpacking algorithm to binpack the user demands into t
 2021-19-01 by Ameer Haj-Ali, Anyscale, Inc.
 
 .. youtube:: BJ06eJasdu4
+
+Next steps
+----------
+
+To get started with Ray Clusters, we recommend that you check out the :ref:`Ray Cluster quick start <ref-cluster-quick-start>`. For more advanced examples of use, you can also refer to the :ref:`full specification for Ray Cluster configuration <cluster-config>`.
```

Comment:
Can you also change the title in quickstart.rst to "Ray Cluster Quick Start"

Reply:
Done!